python/lsst/analysis/ap/apdb.py

```diff
@@ -85,9 +85,9 @@ def connection(self):
         pass
 
     def set_excluded_diaSource_flags(self, flag_list):
-        """Set flags of diaSources to exclude when loading diaSources.
+        """Set flags of DiaSources to exclude when loading DiaSources.
```

Comment:
I think DiaSource should be lowercase in all of these cases: it's not a proper noun, and you're not referring to a class definition. I can be convinced otherwise, but I feel like this is something we should figure out for the dev guide, so we're consistent everywhere.

Reply:
I have called them DIA Sources, DIASources, DiaSources... but diaSources?? ugh. I just want it consistent throughout analysis_ap, and it wasn't.

Reply:
It depends on context: a `diaSource` is a record in the `DiaSource` table. `DIASource` and `DIA Source` aren't the names of any datatype or dataset, they are colloquialisms we use to refer to "sources made by the Difference Imaging Analysis pipeline" (which isn't really a thing either).

In the specific case here, I think both should be lowercase, unless the sentence is phrased `Set diaSource flags to exclude when loading from the DiaSource table.`

Reply:
I at least understand your distinction now 😄 thanks. I suppose we can keep it `diaSource` when we're talking about a single record in a larger table.